openapi/paths/auth/register-org.yml

```diff
@@ -0,0 +1,50 @@
+post:
+  operationId: "RegisterOrg"
```

Comment:
RegisterOrganization

Reply:
updated in a69ce7cbc5d389a5dc3b3f90a56ab0d457f93929